css/style.css

```diff
@@ -6,3 +6,74 @@
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  */
+ .footer {
+    text-align: center;
+ }
+h1 {
+    font-size: 100px;
+  text-align: center;
+  padding-left: 20px;
+  padding-right: 20px;
+  font-size: 18px;
+}
+p{
+    float: left;
+}
+ 
+ 
+ .hello{
+    font-size: 100px;
+  text-align: center;
+  padding-left: 20px;
+  padding-right: 20px;
+  font-size: 18px;
+
+ }
+ .hello1{
+    font-size: 100px;
+  text-align: center;
+  padding-left: 20px;
+  padding-right: 20px;
+  font-size: 18px;
+
+
+ }
```

Comment:
instead of repeating the same stylings for multiple different classes, you can apply the styling to a single class, and apply multiple classes to an element (e.g. `<div class="mystyling hello1">`)